test/Dapr.E2E.Test/ShellCommand.cs

```diff
@@ -0,0 +1,52 @@
+namespace Dapr.E2E.Test
+{
+    using System;
+    using System.Diagnostics;
+    using System.Threading;
+
+    public class ShellCommand
```

Comment:
I should we should just use https://github.com/dotnet/command-line-api/blob/main/src/System.CommandLine/Invocation/Process.cs

Add the dependency, don't copy the code 😆